src/cli/cli.cpp

```diff
@@ -1281,6 +1284,35 @@ void Interpreter::ProcessLinkQuality(int argc, char *argv[])
     AppendResult(error);
 }
 
+#if OPENTHREAD_FTD
+void Interpreter::ProcessPSKc(int argc, char *argv[])
+{
+    ThreadError error = kThreadError_None;
+
+    if (argc == 0)
+    {
+        const uint8_t *pskc = otThreadGetPSKc(mInstance);
+
+        for (int i = 0; i < OT_PSKC_MAX_SIZE; i++)
+        {
+            sServer->OutputFormat("%02x", pskc[i]);
+        }
+
+        sServer->OutputFormat("\r\n");
+    }
+    else
+    {
+        uint8_t pskc[OT_MASTER_KEY_SIZE];
```

Comment:
Thanks for pointing out this. The two variables are of different types, so it's not able to declare once at the function scope. I think declare in different scope help reducing the stack size. Besides, referring `ProcessMasterKey()`, the variable `key` is also declared in both the if and the else. However, they are of different types. What do you think?